### PR TITLE
[Toolchain] Fix SonarLint related issues

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit fixes the problem pointed out in SonarLint.

ONE-vscode-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

**Review by YongseopKim — APPROVED** [review]
LGTM

File: src/Toolchain/ToolchainProvider.ts [hunk 1]
```diff
@@ -161,7 +159,7 @@ export class ToolchainProvider implements vscode.TreeDataProvider<ToolchainNode>
     });
   }
 
-  run(cfg: string) {
+  run(_cfg: string) {
```
Comment (hyunsik-yoon) [inline src/Toolchain/ToolchainProvider.ts:162]: Could you let us know why this change is needed? Thanks in advance.

Comment (jyoungyun) [reply src/Toolchain/ToolchainProvider.ts:162]: This changes came from SonarLint. If this variable is not used in this function, SonarLint suggests removing this variable or adding `_` prefix before the word.

SonarLint warning:
> Remove the unused function parameter "cfg" or rename it to "_cfg" to make intention explicit.sonarlint(typescript:S1172)

File: src/Toolchain/ToolchainProvider.ts [hunk 2]
```diff
@@ -60,12 +61,9 @@ export class ToolchainNode extends vscode.TreeItem {
 export class ToolchainProvider implements vscode.TreeDataProvider<ToolchainNode> {
   tag = this.constructor.name;  // logging tag
 
-  private _onDidChangeTreeData: vscode.EventEmitter<ToolchainNode|undefined|void> =
-      new vscode.EventEmitter<ToolchainNode|undefined|void>();
```
Comment (jyoungyun) [inline src/Toolchain/ToolchainProvider.ts:64]: ToolchainNode|undefined|void

> Replace this union type with a type alias. [+2 locations]sonarlint(typescript:S4323)

<details>
<summary> SonarLint Rule Description </summary>

```
Type aliases should be used (typescript:S4323)
CODE_SMELL  Code Smell  MINOR  Minor
Union and intersection types are convenient but can make code harder to read and maintain. So if a particular union or intersection is used in multiple places, the use of a type alias is recommended.

Noncompliant Code Example
function foo(x:string|null|number) { // Noncompliant
  // ...
}
function bar(x:string|null|number) {
  // ...
}
function zoo(): string|null|number {
  return null;
}
Compliant Solution
type MyType = string | null | number;

function foo(x: MyType) {
  // ...
}
function bar(x: MyType) {
  // ...
}
function zoo(): MyType {
  return null;
}
```

</details>

File: src/Toolchain/ToolchainProvider.ts [hunk 3]
```diff
@@ -60,12 +61,9 @@ export class ToolchainNode extends vscode.TreeItem {
...
-  readonly onDidChangeTreeData?: vscode.Event<ToolchainNode|undefined|void> =
-      this._onDidChangeTreeData.event;
-
-  constructor() {}
```
Comment (jyoungyun) [inline src/Toolchain/ToolchainProvider.ts:68]: > Unexpected empty constructor.sonarlint(typescript:S1186)

**Review by hyunsik-yoon — APPROVED** [review]
